pinnwand/utility.py

```diff
@@ -26,6 +26,41 @@ def list_languages() -> Dict[str, str]:
     return dict(sorted(lexers.items(), key=lambda x: x[1]))
 
 
+GUESS_LANG_OVERRIDE_MAP = {
+    'python2': 'python'
+}
+
+def guess_language(raw: str, filename: str) -> str:
+    # Shorten the raw text to 1024 bytes, increases chance to get matching right
+    raw = raw[:1024]
+
+    # Guess a lexer based on filename and raw text first
+    if filename:
+        try:
+            return guess_lexer_for_filename(filename, raw).aliases[0]
+        except ValueError:
+            pass
+
+    # If that didn't work guess lexer just by looking at the raw text
+    try:
+       language = guess_lexer(raw).aliases[0]
+    except ValueError:
+        # If no lexer was detected, fallback to plain text.
+        return 'text'
+
+    # These are odd lexers that match far too often, so exclude them.
+    if language in ('mime', 'tsql'):
+        return 'text'
+
+    # Finally check for language overrides
+    try:
```

Comment:
Also just a consistency thing but I think I mostly use ``.get`` in the codebase; so ``return GUESS_LANG_OVERRIDE_MAP.get(language, language)`` might suffice?

Reply:
Adressed